tests/bluetooth/bsim_bt/edtt_ble_test_app/gatt_test_app/prj.conf

```diff
@@ -24,8 +24,10 @@ CONFIG_BT_L2CAP_RX_MTU=512
 CONFIG_BT_L2CAP_TX_MTU=512
 CONFIG_BT_CTLR_RX_BUFFERS=3
 CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_EATT=n
```

Comment:
CC: @heri-developer

Reply:
Just a remimder here, test shall not assume handle positions are static on the database, if a handle is required for a test perform a lookup into the database to resolve the handle at runtime.

Reply:
Could you maybe create an issue for EDTT (https://github.com/EDTTool/EDTT) to describe what is broken/needs enhancement in EDTT when enabling EATT?

Reply:
I though this was quite clear by now, anyway autopts might needs adjusting as well.

Reply:
The Bluetooth-SIG GATT test database(s) does use static assigned handles, that is an essential part of the tests. Until they decide to change test-strategy, we need to be able to support that.

Reply:
Comments in this PR will be forgotten as soon as it is merged. An issue will remind us what needs fixing in the future.

Reply:
> The Bluetooth-SIG GATT test database(s) does use static assigned handles, that is an essential part of the tests. Until they decide to change test-strategy, we need to be able to support that.

I guess you are refering to the fact the you enter the handles in pixit or something like that, but we don't have to follow how PTS does things here since EDTT just emulates the tests we can actually inform the handles at runtime, so note that the purpose of these tests are to tests the procedures not the actual database layout since GATT don't actually have a concept of fixed/pinned services, the order is undefined.

Reply:
https://github.com/EDTTool/EDTT/issues/16

Reply:
No - I'm referring to the official GATT Qualification Test DB that can be found here: https://www.bluetooth.org/docman/handlers/DownloadDoc.ashx?doc_id=229720.

Reply:
@heri-developer isn't selecting a SIG-provided database optional in the GATT IXIT, i.e. it's still possible to qualify stacks without such a database? I'm looking e.g. at GATT:P4.1 and GATT:P4.2 and both have a comment "Alternatively, a SIG-provided database may be used."

Reply:
Ive actually end up dropping this since EATT is no longer enabled by default.